### ISSUE TITLE
Links to code.AMO should redirect to the corresponding file and line in VS Code

### ISSUE THREAD

**Issue by abhn** [mozilla/assay#31]
We have documentation that points to lines of code in particular versions of add-ons, such as `https://code.addons.mozilla.org/en-US/browse/123/versions/123/?allowSlowPages=true&path=main.js#I123`

We'd like to set up a redirect such that this link is captured by VS code and it downloads and opens the said file and line number.

**Comment by eviljeff** [comment]
> We have documentation 

Can you link to this documentation?  We could just change it.

**Comment by abhn** [comment]
I meant internal documentation in various places like google docs.

> We could just change it.

How would we go about this? As in, would the change be a manual one, or can we set up a redirect such that it turns into a VSC link that can open the right file and line (after downloading the add-on version, if required).

**Comment by eviljeff** [comment]
> I meant internal documentation in various places like google docs.
> 
> > We could just change it.
> 
> How would we go about this? As in, would the change be a manual one

Yes, we literally change the links.  It's not like these are externally accessible urls spread across the internet.
